include/libtorrent/span.hpp

```diff
@@ -60,10 +60,12 @@ namespace libtorrent
 		span(U (&arr)[N])
 			: m_ptr(&arr[0]), m_len(N) {}
 
+#if (defined __GNUC__ || (defined _MSC_VER && _MSC_VER >= 1900))
```

Comment:
not sure about this. libtorrent would fail to build unless that condition is true, right?